app/forms/destination/schedule_rate_changes_form.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Destination
+  class ScheduleRateChangesForm < ApplicationForm
+    attribute :apply_time, :date
+    attribute :initial_interval, :integer
+    attribute :initial_rate, :decimal
+    attribute :next_interval, :integer
+    attribute :next_rate, :decimal
+    attribute :connect_fee, :decimal
+    attribute :ids_sql, :string
+
+    validates :initial_rate, :next_rate, :connect_fee, numericality: true
```

Comment:
Can they be negative? I mean what happens if we enter '-1.0'?

Reply:
Those validations just copied from DestinationNextRate model. And already we allow to create such next_rates with negative initial rate, next rate and connect fee. But i agree what it's looks wrong. Maybe need add validation for DestinationNextRate.

Reply:
let's do this in the separated issue